build-tools/mono-runtimes/ProfileAssemblies.projitems

```diff
@@ -192,6 +192,10 @@
     <MonoTestAssembly Include="monodroid_corlib_test.dll">
       <SourcePath>corlib</SourcePath>
     </MonoTestAssembly>
+    <MonoTestAssembly Include="monodroid_corlib_xunit-test.dll">
+      <SourcePath>corlib</SourcePath>
+      <TestsType>xunit</TestsType>
```

Comment:
Should we add `%(MonoTestAssembly.TestsType)` to everything, and use `nunit` for the existing ones?

Also, shouldn't this be singular, i.e. `TestType` not `TestsType`?

Reply:
Changed to `TestType` but regarding adding `nunit` to the other items, I don't think it makes sense. This entire thing is a temporary measure until we have xUnit equivalent of `make test` in Mono and adding the `nunit` string to all the other items would generate unnecessary noise since it wouldn't be used anywhere anyway.